### PR TITLE
Reorganised the document

### PR DESCRIPTION
**Pull request by voroyam**
- removed unnecessary paragraphs 
- streamlined the document for installation and configuration

### CODE REVIEW

File: modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc [hunk 1]
```diff
@@ -144,14 +67,19 @@ For example, to run it every hour at 47 minutes past the hour, add the following
...
 
 image:apps/files_antivirus/antivirus-app.png[image]
 
-TIP: The Anti-Virus app can also be downloaded, installed, and enabled manually.
+--------
+
+To install the App via occ command, execute:
+
+----
+occ market install files_antivirus
```
Comment (voroyam) [inline modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc:81]: need occ market:install files_antivirus

File: modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc [hunk 2]
```diff
@@ -411,3 +339,32 @@ Then follow the process outlined in xref:default-ruleset[Update An Existing Rule
...
+
+== Things To Note
+
+. Files are checked when they are uploaded or updated (whether because they were edited or saved) but _not_ when they are downloaded.
+. ownCloud doesn't support a cache of previously scanned files.
+. If the app is either not configured or is misconfigured, then it rejects file uploads.
+. If ClamAV is unavailable, then the app rejects file uploads.
+. A file size limit applies both to background jobs and to file uploads.
+. We recommend that you enable verbose logging in both `clamd.conf` and `freshclam.conf` until you get any kinks with your ClamAV installation worked out.
+. After installing ClamAV and the related tools, you will have two configuration files: `/etc/freshclam.conf` and `/etc/clamd.d/scan.conf`.
```
Comment (voroyam) [inline modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc:370]: put one line above

**Review by enbrnz — APPROVED** [review]
We talked through all the changes, looking good to me

**Review by enbrnz — COMMENTED** [review]
LGTM

**Pull request closed without merging**